### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by yhoogstrate**
Hey @ndaniel ,

I have been working with FusionCatcher and I changed a few lines of your code. In generate_banned.py I have added three 'fusion-genes' that were in my top list and were obviously no real fusion gene: MALAT1-RPPH1. MALAT1-KLK3 and SLCA45A3-ELK4 (I saw that ELK4-SLCA45A3 was already part of your code). If you need more details I can provide it. I also sorted the list and ran uniq on it, to remove some duplicate entries. I hope you like the change and you could test it. 

Thanks in advance,

Youri

### CODE REVIEW

**Comment by yhoogstrate** [comment]
Hey @ndaniel, thanks for looking into this. The reason I added it to your code is because on of my top hits is:

```
SLC45A3 ELK4    adjacent,known,cosmic,chimerdb2,cgp,ticdb,healthy,cacg,hpa,banned,10K<gap<100K,readthrough      0       42      10      58      BOWTIE+BLAT;BOWTIE+STAR 1:205659489:-   1:205623891:-   ENSG00000158715 ENSG00000158711                 TGCTCCCACCTCCACCCGCGCTCTGCGGGGCCTCTGCCTGTGATGTCTCC*CTCATTGCTATGGACAGTGCTATCACCCTGTGGCAGTTCCTTCTTCAGCT   CDS(truncated)/UTR              
SLC45A3 ELK4    adjacent,known,cosmic,chimerdb2,cgp,ticdb,healthy,cacg,hpa,banned,10K<gap<100K,readthrough      0       42      9       62      BOWTIE;BOWTIE+BLAT;BOWTIE+STAR  1:205661861:-   1:205623891:-   ENSG00000158715 ENSG00000158711 ENSE00003644968 ENSE00003720094 TGCAGATCCTGCCCTACACACTGGCCTCCCTCTACCACCGGGAGAAGCAG*CTCATTGCTATGGACAGTGCTATCACCCTGTGGCAGTTCCTTCTTCAGCT   CDS(truncated)/UTR              
SLC45A3 ELK4    adjacent,known,cosmic,chimerdb2,cgp,ticdb,healthy,cacg,hpa,banned,10K<gap<100K,readthrough      0       42      1       59      BOWTIE+BLAT;BOWTIE+STAR 1:205664485:-   1:205623891:-   ENSG00000158715 ENSG00000158711                 TCTGCTGCTGGAAGTGGGGGTAGAGGAGAAGTTCATGACCATGGTGCTGG*CTCATTGCTATGGACAGTGCTATCACCCTGTGGCAGTTCCTTCTTCAGCT   CDS(truncated)/UTR  
```

While I have:
```
~/src/fusioncatcher/fusioncatcher/data/current$ grep ENSG00000158715 banned.txt 
ENSG00000158711	ENSG00000158715
```

I am using v0.99.6a. For now, enjoy your holidays! If I am able to figure out what happend I will post updates here.

**Comment by ndaniel** [comment]
The fusion SLC45A3-ELK4 (and ELK4-SLC45A3) is really challenging fusion. In the scientific literature are articles which associate this fusion with some cancers/disease. On the other hand out there are also other scientific articles which show that the opposite of this, which is that this fusion is found in healthy people and it is not associated with disease/cancers. In this kind of cases, FusionCatcher does not try to prove right or wrong any of these articles and it simply shows to the users, using the labels "known,banned" that this fusion is known to be associated with disease/cancer and in the same time it has been found also in healthy tissues/people. Basically, FusionCatcher lets the user decide what to do with this fusion. Also the label "known" has priority of label "banned", which means that if a fusion is labeled as "known" it will not be removed or skipped from the analysis even if it is labeled as "banned".  Another challenge here regarding this fusion is that it may happen that some research includes this fusion  SLC45A3-ELK4 (because there are articles which shows that it found in cancer) in his/her panel of fusion genes which is used for evaluating several fusion finders, and if FusionCatcher would not report it then it worsen the FP and TP rates.

As conclusion, the behavior of FusionCatcher regarding **ambiguous** SLC45A2-ELK4 fusion is as expected (that is report this fusion with labels "known,banned").

Personally, it looks to me tat SLC45A2-ELK4 fusion is false positive and it has nothing to do with the disease/cancer and sooner or later this will be proven in scientific literature.

File: bin/blat-filter-fusion.py [hunk 1]
```diff
@@ -48,17 +48,14 @@
...
-    r = False
     lin = txt.rstrip('\r\n').split('\t')
-    if ( lin and
+    return ( lin and
         (float(lin[0])/float(lin[10])) >= 0.70 and  # number of matches / length of query sequence
         lin[17] == '2' and  # blockCount
         lin[6] == '1' and   # number inserts in target = tNumInserts
         (lin[4] == '0' or (lin[4] == '1' and int(lin[5]) <= 3)) and  # number inserts in query
         min(map(int,lin[18][:-1].split(','))) > 9  # blockSizes
         ):
```
Comment (yhoogstrate) [inline bin/blat-filter-fusion.py]: I think my semicolon should be removed

Comment (ndaniel) [inline bin/blat-filter-fusion.py]: One comment regarding the generate_banned.py. MALA1-KLK3 has been added but I guess that you meant MALAT1-KLK3. Right?

Comment (yhoogstrate) [inline bin/blat-filter-fusion.py]: Yes, that's what I meant. Although I am still digging through the data I found quite some more candidates for another PR. Thanks anyway @ndaniel :+1:

**Pull request merged**